### ISSUE TITLE
Android UI Issue

### ISSUE THREAD

**Issue by FroggyFriend** [Bionus/imgbrd-grabber#2328]
**Bug description**
I was looking at some images on the Grabber Android Version 7.5.1 and it gave results for the images that would require me to slide down to see, but it wouldn't let me scroll down, it was kinda stuck.
I dont know if this is an app issue or just a general bug with Android but I figured I'd still report on it

**Steps to reproduce**
(This has happened to me multiple times but I cant find a persistent way to make it happen so you may not reproduce it)

1. Search any tag on Android Grabber 7.5.1 that displays enough images so that you have to scroll down to see the results
2. Try to Scroll down

Grabber Version 7.5.1
Android Version 9

**Comment by Bionus** [comment]
Yes I had this occur to me as well a few times, like the scroll area is not properly going to the bottom. 🤔 
I'll have a look
